content/contributing/documentation/rst_guidelines.rst

```diff
@@ -1,162 +1,1365 @@
+
+         Use relative links for internal URLs
+         ------------------------------------
+
+         Please refer to the :ref:`<contributing-rst-hyperlinks-guidelines>` section to learn more
+         about :ref:`relative links <contributing-rst-relative-links>`.
+
+.. _contributing-rst-doc-page-hyperlinks:
+
+Documentation page hyperlinks
+-----------------------------
+
+| The ``doc`` markup allows referencing a documentation page wherever it is in the file tree through
+  a relative file path.
+| There are two ways to use the markup, both using the ``doc`` markup:
+
+
+#. ``:doc:`path_to_doc_page``` creates a hyperlink to the documentation page with the title of the
+   page as label.
+#. ``:doc:`label <path_to_doc_page>``` creates a hyperlink to the documentation page with the given
+   label.
+
+.. list-table::
+   :class: o-showcase-table
+
+   * - Please refer to `the Invoicing documentation <https://example.com/doc/accounting/invoices.html>`_
+       and to `Send a pro-forma invoice <https://example.com/doc/sales/proforma.html>`_.
+
+   * - .. code-block:: text
+
+          Please refer to :doc:`this documentation <customer_invoices>` and to
+          :doc:`../sales/sales/invoicing/proforma`.
+
+.. important::
+   :ref:`Use relative links <contributing-rst-relative-links>` for documentation page hyperlinks.
+
+.. _contributing-rst-download:
+
+File download hyperlinks
+------------------------
+
+The ``download`` markup allows referencing files (that are not necessarily :abbr:`RST
+(reStructuredText)` documents) within the source tree to be downloaded.
+
+.. list-table::
+   :class: o-showcase-table
+
+   * - Download this `module structure template
+       <https://example.com/doc/odoosh/extras/my_module.zip>`_ to start building your module in no
+       time.
+
+   * - .. code-block:: text
+
+          Download this :download:`module structure template <extras/my_module.zip>` to start
+          building your module in no time.
+
+.. _contributing-rst-images:
+
+Images
+======
+
+The ``image`` markup allows inserting images in a document.
+
+.. list-table::
+   :class: o-showcase-table
+
+   * - .. image:: rst_guidelines/create-invoice.png
+          :alt: Create an invoice.
+
+   * - .. code-block:: text
+
+          .. image:: rst_guidelines/create-invoice.png
+             :alt: Create an invoice.
+
+.. tip::
+   - Images should generally be aligned to the left, which is the default behavior. Use the `align`
+     parameter to change the alignment, e.g., ``:align: center``.
+   - Use the `scale` parameter to scale the image, e.g., ``:scale: 75%``.
+   - You can also use the ``figure`` markup instead of ``image`` if you want to add captions to
+     your screenshot.
+
+.. seealso::
+   :ref:`Content guidelines for images <contributing-content-images>`
+
+.. _contributing-rst-alert-blocks:
+
+Alert blocks (admonitions)
+==========================
+
+.. _contributing-rst-seealso:
+
+See also
```

Comment:
Should we mention that it's okay to include more text than _just_ the link itself?

Reply:
This works, but what I had in mind was text _outside_ of the the hyperlink altogether. This is a bad example because the text outside of the link is irrelevant but something like this:
![image](https://github.com/user-attachments/assets/4c5cee4a-426f-4da9-b4d8-6be84cdb3774)

Reply:
I kind of think it isn't really necessary for seealsos in most cases. For instance, in your example above, I would personnally only have "Contact form configuration documentation" (with the hyperlink on the whole thing) in the seealso.